backend/de.metas.purchasecandidate.base/src/main/java/de/metas/purchasecandidate/material/event/PurchaseCandidateRequestedHandler.java

```diff
@@ -122,12 +117,14 @@ public void handleEvent(@NonNull final PurchaseCandidateRequestedEvent event)
 				.userElementString7(event.getUserElementString7())
 				.build();
 
+		final ZonedDateTime datePromised = TimeUtil.asZonedDateTime(materialDescriptor.getDate());
 		final PurchaseCandidate newPurchaseCandidate = PurchaseCandidate
 				.builder()
 				.groupReference(DemandGroupReference.EMPTY)
 				.vendorId(vendorProductInfos.getVendorId()) // mandatory
 				.vendorProductNo(vendorProductInfos.getVendorProductNo()) // mandatory
-				.purchaseDatePromised(TimeUtil.asZonedDateTime(materialDescriptor.getDate())) // dateRequired
+				.purchaseDatePromised(datePromised) // dateRequired
+				.purchaseDateOrdered(getPurchaseDateOrderedOrNull(datePromised, event.getProductPlanningId())) // dateRequired
```

Comment:
* imho "// dateRequired" is no longer relevant
* consider renaming getPurchaseDateOrderedOrNull to computePurchaseDateOrderedOrNull

Reply:
done